src/robot.cpp

```diff
@@ -408,6 +419,33 @@ void Robot::execute()
   }
 }
 
+geometry_msgs::Pose Robot::getTarget()
+{
+  std::string planning_group_name = config_->getFeatureManipulation().getActivePlanningGroup();
+  
+  auto group_it = planning_groups_.find(planning_group_name);
+  TEMOTO_INFO(planning_group_name.c_str());
+
+  geometry_msgs::Pose current_pose;
+  
+  if (group_it != planning_groups_.end())
+  {    
+    current_pose = group_it->second->getCurrentPose().pose;
+  }
+  else 
```

Comment:
In the `else` case the returned message will be empty, which might not be a good way to indicate errors. Please write write a `//TODO: This section has to utilize temoto error management system` in the `else` block and for now leave as it is.